beancount/projects/export.py

```diff
@@ -252,6 +253,15 @@ def write_table(table: Table, outfile: str):
     writer.writerows(table.rows)
 
 
+# python 3.15 will make utf-8 mode enabled by default
+# so use a locale encoding for backward compatibility
+if sys.version_info < (3, 10):
+    __output_encoding = None
```

Comment:
Why do you prefix the variable name with two underscores? I would not use even one, but two just look odd.

Reply:
According to the style guide, globals should be in caps, e.g. OUTPUT_ENCODING or _OUTPUT_ENCODING

Reply:
done